modules/tinymce/src/core/main/ts/fmt/MatchFormat.ts

```diff
@@ -47,7 +47,7 @@ const matchParents = (editor: Editor, node: Node, name: string, vars: FormatVars
   });
 
   // Do an exact check on the similar format element
-  return matchNode(editor, node, name, vars);
+  return !!matchNode(editor, node, name, vars);
```

Comment:
This should be harmless, as we were only looking at the output of matchParents as a boolean.

That said, probably could use:
```suggestion
  return matchNode(editor, node, name, vars) !== undefined;
```

Reply:
I think this is fine for now, as it's doing the same a couple lines above. If we do make apply the suggestion, please make sure to do it in both places.

Reply:
Should this be:
`return !Type.isUndefined(matchNode(editor, node, name, vars));`
or
`return Type.isUndefined(matchNode(editor, node, name, vars)) === false;`

Reply:
Yeah, if we do change it, it should be one of those.